Cargo.toml

```diff
@@ -11,3 +11,4 @@ gio = "0.18.4"
 glib = "0.18.5"
 gsettings-macro = "0.1.20"
 swayipc = "3.0.1"
+zbus = "4.2.2"
```

Comment:
concern: This version will be too recent for `debian-testing` (https://packages.debian.org/testing/rustc)

```
$ volsungr 1.71.1 zbus
Searching zbus versions compatible with rust 1.71.1...
Latest compatible version of zbus = "3.15.2"
```

Reply:
Good find. Will change and test with `3.15.2`. Or do we need to go even lower (for supporting older releases)?

Reply:
I think that's the earliest version, and what we'd decided upon last year as minimum rust version.  Looks like Jammy is on a newer `rustc`: https://packages.ubuntu.com/jammy/rustc